images/eh-expansion-evaluator/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM ubuntu:20.04
+ARG DEBIAN_FRONTEND=noninteractive
+
+ARG VERSION=${VERSION:-eb982ae41f83aaf44d4a8142980f7a41ac33e979}
+
+# Install Git to clone the repository
+RUN apt-get update && \
```

Comment:
Needs a Python(3) installation, this might be a model for that:
https://github.com/populationgenomics/images/blob/main/images/expansionhunter/Dockerfile#L6-L9

Reply:
And on top of that Python installation you need to pip install the required libraries
https://github.com/oscarlr-TRs/EH-expansion-evaluator#requirements